SDL_UDP_Networking/SDL_net_test/UDP_Connection.h

```diff
@@ -104,6 +110,8 @@ struct UDPConnection
 	// Send data. 
 	bool Send(const std::string &str)
 	{
+		if (str == "quit")
```

Comment:
Try to avoid hard coding strings in the body of your code, make a constant instead.